Sensus.Shared/UI/PendingScriptsPage.cs

```diff
@@ -19,6 +19,9 @@
 using Sensus.UI.Inputs;
```

Comment:
Clean up the code below to eliminate changes.

Reply:
Done, sorry about that.  I'm used to VS automatically removing unused imports.  I guess the free version doesn't do that.